frontend/src/features/PriorNotification/components/PriorNotificationList/index.tsx

```diff
@@ -232,8 +241,18 @@ export function PriorNotificationList({ isFromUrl }: PriorNotificationListProps)
         </Body>
       </Page>
 
-      {isPriorNotificationCardOpen && <PriorNotificationCard />}
-      {isPriorNotificationFormOpen && <PriorNotificationForm />}
+      {isPriorNotificationCardOpen && (
```

Comment:
```suggestion
      {priorNotificationOpened === PriorNotificationSidebarType.CARD && (
```
J'aurais bien vu un enum pour éviter 3 states et clarifier la compréhension qu'un seul state doit être à `true`. Mais c'est pas vital